app/modules/listings/components/listing_element_show.svelte

```diff
@@ -0,0 +1,125 @@
+<script lang="ts">
+  import { createEventDispatcher } from 'svelte'
+  import { isNonEmptyPlainObject } from '#app/lib/boolean_tests'
+  import Flash from '#app/lib/components/flash.svelte'
+  import { icon } from '#app/lib/icons'
+  import EntityImage from '#entities/components/entity_image.svelte'
+  import AuthorsInfo from '#entities/components/layouts/authors_info.svelte'
+  import Ebooks from '#entities/components/layouts/ebooks.svelte'
+  import EntityTitle from '#entities/components/layouts/entity_title.svelte'
+  import Infobox from '#entities/components/layouts/infobox.svelte'
+  import Summary from '#entities/components/layouts/summary.svelte'
+  import ListingElementComment from '#modules/listings/components/listing_element_comment.svelte'
+  import { I18n } from '#user/lib/i18n'
+
+  export let element, entity, isCreatorMainUser
+
+  const dispatch = createEventDispatcher()
+  const { type, claims } = entity
+
+  let flash
+</script>
+
+<div class="listing-element-show">
+  <div class="entity-type-label">
+    {I18n(type)}
+  </div>
+  <EntityTitle
+    {entity}
+    hasLinkTitle={true}
+  />
+  <div class="entity-info-row">
+    {#if isNonEmptyPlainObject(entity.image)}
+      <EntityImage
+        {entity}
+        size={256}
+      />
+    {/if}
+    <div class="entity-infobox">
+      <AuthorsInfo {claims} />
+      <Infobox
+        {claims}
+        entityType={type}
+        shortlistOnly={true}
+      />
+      <Ebooks {entity} />
+    </div>
+  </div>
+
+  <Summary {entity} />
+
+  {#if element.comment || isCreatorMainUser}
+    <ListingElementComment
+      {isCreatorMainUser}
+      bind:element
+      bind:flash
+    />
+  {/if}
```

Comment:
[non-blocking, could be adjusted later]
I think it would make sense to display the comment just below the title, as, when it exists, it's the most important part of that page. I think that's already true when opening the element modal from the list context, but will be even more when opening it from the entity context: if you want to read what different users have to say about the entity, you are not primarily interested in reading the entity details over and over

Reply:
Attempt done in b21c68acf